geocodio/__init__.py,sha256=rRiQrPiHyJQqUtaLhVwzmle3pZHdrFIDoSBOkOkSGbI,385
geocodio/__pycache__/__init__.cpython-38.pyc,,
geocodio/__pycache__/client.cpython-38.pyc,,
geocodio/__pycache__/data.cpython-38.pyc,,
geocodio/__pycache__/exceptions.cpython-38.pyc,,
geocodio/client.py,sha256=m5isZcOmIQWjNyucHQpmRQBII6TS5fhqz3QvoZw6is8,8941
geocodio/data.py,sha256=byJ9bipvI6ug5uvAyBbfnVct2Q4ChuBPSAVRgW4vXWk,4230
geocodio/exceptions.py,sha256=ogv-rAfdiF2Pd4KbPyGpTcExuD0s2VJw6nrax2g4KV0,438
pygeocodio-0.11.1.dist-info/AUTHORS.rst,sha256=a9AvBcbMymw16ROFgO7cfSrAh0JU2cGwflZIjTF7izg,686
pygeocodio-0.11.1.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pygeocodio-0.11.1.dist-info/LICENSE,sha256=AkR3PsGAEmG3F_c2uMHA3JvDoHimA1rYK32Ra_-LAc8,1476
pygeocodio-0.11.1.dist-info/METADATA,sha256=S4i6afQ9E7bS-9XeolfuT4eHAdkmteMYTvydHcNeSKY,7324
pygeocodio-0.11.1.dist-info/RECORD,,
pygeocodio-0.11.1.dist-info/WHEEL,sha256=S8S5VL-stOTSZDYxHyf0KP7eds0J72qrK0Evu3TfyAY,92
pygeocodio-0.11.1.dist-info/top_level.txt,sha256=mC-G7vXzLU9WCuQIA192zOi4k__HtnhAI--OvWSoMZs,9
